act_5.1/ACT_5.1_HASHING.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
```

Comment:
Cuando corro este código se modifica la tabla por el método cuadrático 2 veces?
<img width="359" alt="Screenshot 2023-11-26 at 3 55 51" src="https://github.com/casrhub/Data-Structures-and-Algorithms/assets/115209085/fcd64988-ac8e-4764-841c-3e922e13bb19">

Reply:
nimba apache solo una vez con cuadrático y la otra con encadenamiento

Reply:
ya veo... es que salen 3:
<img width="320" alt="Screenshot 2023-11-26 at 4 02 06" src="https://github.com/casrhub/Data-Structures-and-Algorithms/assets/115209085/c25282b5-6a1d-42ae-a664-e1447cad6e96">

Reply:
esta raro, pero en mi local corre bien no te preocupes, mañana nos depositan sin problema el dinero.